lib/about.ts

```diff
@@ -1,5 +1,8 @@
-export const About = function (picturesSource, picturesLicense) {
-  this.render = function render(d) {
+import { CanRender } from "./container";
+
+export const About = function (picturesSource: string, picturesLicense: string): CanRender {
+  function render(d: HTMLElement) {
+    let _ = window._;
```

Comment:
This looks like a global function → I would suggest to export it from a module and add type specs.